apps/storybook/src/Table.stories.tsx

```diff
@@ -314,9 +314,15 @@ export const Sortable: Story<Partial<TableProps>> = (args) => {
         columns: [
           {
             id: 'name',
-            Header: 'Name',
+            Header: 'Name (sort asc first, default)',
```

Comment:
That's fine with me. I also spelled out 'ascending' and 'descending' just to make it clearer

Reply:
I actually think maybe `Asc First` and `Desc first` are enough. Because even the video in the linked issue had that.

![image](https://user-images.githubusercontent.com/45748283/185211602-9961213b-001c-43ad-8d34-d974b2adf668.png)

Reply:
I shortened ascending and descending

Reply:
Yes, making it to just `Asc` and `Desc` would be great. But about writing `sort`, I think that might be redundant, Because `Asc First` usually means sorted asc first. Also the name of the story is `Sortable` so it would also hint at the same.

But I don't think it might matter that much  😅 Any of those seems okay to me! It was just something small I noticed so I thought I'd mention it

Reply:
Well, I think I'll leave it as it is now then.